.github/workflows/draft-release.yml

```diff
@@ -51,17 +51,17 @@ jobs:
         with:
```

Comment:
I can't comment on the line, but L40 above has:

```node
echo "new_version=${NEW_VERSION}" >> $GITHUB_OUTPUT
```

I think we want to put that in the `env` instead:

```node
echo "NEW_VERSION=${NEW_VERSION}" >> $GITHUB_ENV
```

Then we can access it in subsequent action-script as `process.env.NEW_VERSION` without needing to escape the value.